index.html

```diff
@@ -224,21 +243,32 @@ <h4>Skills</h3>
         </section>
 
         <footer>
-            <div class="remarks">
-                <p>
-                    I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
-                </p>
+            <div class="last-part">
+                <div class="remarks">
+                    <p>
+                        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                    </p>
+                </div>
+    
```

Comment:
- [ ] Kindly consider the following

- Your design
<img width="962" alt="image" src="https://github.com/Wils-FOURR-speed/My-portfolio/assets/41163571/9b4ec2b6-5886-417f-bff6-d31c04c6b390">

- Figma design
<img width="392" alt="image" src="https://github.com/Wils-FOURR-speed/My-portfolio/assets/41163571/f337d73f-df67-4d6a-80ac-2d78b5dae7a1">


- Kindly ensure that the screen accommodates three cards, like it is in the original design.
- Kindly increase the spaces  outside the cards.
- Kindly reduce the space between the cards, both horizontally and vertically.
- The cards aren't the same width, I noticed that the first card is bigger that the second, I'm sure you can agree to that.